runtimes/huggingface/README.md

```diff
@@ -55,6 +55,19 @@ MLSERVER_MODEL_HUGGINGFACE_OPTIMUM_MODEL=true
 ```
 ````
 
+### Loading models
+#### Local models
+It is possible to load a model from a local path into a HuggingFace pipeline by leaving both `parameters.uri` and `parameters.extra.pretrained_model` empty in the `model-settings.json`.
+In this case, the HuggingFace runtime will populate `parameters.uri` to have the value of the model folder where `rclone` will have
+downloaded the model artefacts folder from the specified in the UI remote location.
```

Comment:
I wouldn't reference `rclone` here TBH. Instead, I would just keep it simple and point out that if you want to load a local model you can use the `parameters.uri` field to point to where your local artefact is located.

Main reason for this is that rclone is an implementation detail from Seldon Core - i.e. MLServer doesn't know anything about that. In fact, MLServer can be used outside Seldon Core (in which case, rclone wouldn't be present).

Reply:
Good point. I will leave it out. Thanks!